### ISSUE TITLE
Crashed during first run

### ISSUE THREAD

**Issue by agemooij** [telepresenceio/telepresence#1077]
### What were you trying to do?

First time test of telepresence against an Openshift 3.11 cluster

### What did you expect to happen?

Telepresence starts up its shell and lets me play around with it

### What happened instead?

It crashed. Here's the first few lines of output before the crash:

```
T: Warning: kubectl 1.15.0 may not work correctly with cluster version 1.11.0+d4cacc0 due to the version discrepancy. See https://kubernetes.io/docs/setup/version-skew-policy/ for more information.

T: Failed to find OpenShift deploymentconfig None:
T:   No resources found.
T: Error from server (NotFound): deploymentconfigs.apps.openshift.io "None" not found
T: Will try regular Kubernetes Deployment.
T: How Telepresence uses sudo: https://www.telepresence.io/reference/install#dependencies
T: Invoking sudo. Please enter your sudo password.
Password:
T: Starting proxy with method 'vpn-tcp', which has the following limitations: All processes are affected, only one telepresence can run per machine, and you can't use other VPNs. You may need to add cloud hosts and headless services with
T:  --also-proxy. For a full list of method limitations see https://telepresence.io/reference/methods.html
T: Volumes are rooted at $TELEPRESENCE_ROOT. See https://telepresence.io/howto/volumes.html for details.
T: Starting network proxy to cluster using new Deployment telepresence-1563282105-445678-64701
```

### Automatically included information

Command line: `['/usr/local/bin/telepresence']`
Version: `0.101`
Python version: `3.7.3 (default, Mar 27 2019, 09:23:15) 
[Clang 10.0.1 (clang-1001.0.46.3)]`
kubectl version: `Client Version: v1.15.0 // Server Version: v1.11.0+d4cacc0`
oc version: `oc v3.11.16 // kubernetes v1.11.0+d4cacc0 // features: Basic-Auth //  // Server https://zandvoort.lightbend.com:443 // openshift v3.11.98 // kubernetes v1.11.0+d4cacc0`
OS: `Darwin Ultimate-Laptop-VI.local 18.6.0 Darwin Kernel Version 18.6.0: Thu Apr 25 23:16:27 PDT 2019; root:xnu-4903.261.4~2/RELEASE_X86_64 x86_64`

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 136, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 60, in main
    remote_info = start_proxy(runner)
  File "/usr/local/bin/telepresence/telepresence/proxy/__init__.py", line 102, in start_proxy
    run_id=run_id,
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 153, in get_remote_info
    runner, deployment_name, deployment_type, run_id=run_id
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 100, in get_deployment_json
    )["items"][0]
IndexError: list index out of range

```

Logs:

```
88b90ec4305d2fc03
  33.1  12 | No resources found
  33.1 TEL | [12] ran in 0.53 secs.
  33.1 TEL | [13] Running: oc --context default/zandvoort-lightbend-com:443/age --namespace default run --restart=Always --limits=cpu=100m,memory=256Mi --requests=cpu=25m,memory=64Mi telepresence-1563282105-445678-64701 --image=datawire/telepresence-k8s:0.101 --labels=telepresence=ee82a056c8664bd88b90ec4305d2fc03
  33.6  13 | deploymentconfig.apps.openshift.io/telepresence-1563282105-445678-64701 created
  33.6 TEL | [13] ran in 0.49 secs.
  33.6 TEL | END SPAN deployment.py:97(create_new_deployment)    1.0s
  33.6 TEL | BEGIN SPAN remote.py:151(get_remote_info)
  33.6 TEL | BEGIN SPAN remote.py:78(get_deployment_json)
  33.6 TEL | [14] Capturing: oc --context default/zandvoort-lightbend-com:443/age --namespace default get deployment -o json --selector=telepresence=ee82a056c8664bd88b90ec4305d2fc03
  34.1 TEL | [14] captured in 0.47 secs.
  34.1 TEL | END SPAN remote.py:78(get_deployment_json)    0.5s

```

**Comment by ark3** [comment]
See also #1010

**Comment by ark3** [comment]
I believe this was fixed by #1404.
